templates/comment_form/email_field.twig

```diff
@@ -1,5 +1,14 @@
-<div class="form-group animated-label mb-0">
-	{% set your_email = __( 'Your Email', 'planet4-master-theme' ) %}
-	<input type="email" class="form-control" id="comments-email-input" name="email" placeholder="{{ your_email }}" required />
+<div class="form-group animated-label comment-email">
+	{% set placeholder = __( 'Your Email', 'planet4-master-theme' ) %}
+	{% set commenter_email = function('wp_get_current_commenter').comment_author_email ?? '' %}
```

Comment:
Same as earlier, this is the fix for the case where the "Cookies opt-in" checkbox is enabled in the backend and checked by the commenter